frontend/src/components/VideoCall/VideoFrontend/components/MenuBar/MenuBar.tsx

```diff
@@ -82,6 +84,17 @@ export default function MenuBar(props: { setMediaError?(error: Error): void }) {
             <Grid container justify="center">
               <ToggleAudioButton disabled={isReconnecting} setMediaError={props.setMediaError} />
               <ToggleVideoButton disabled={isReconnecting} setMediaError={props.setMediaError} />
+              <FormControlLabel
+                control={
+                  <Switch
+                    checked={appState.enableInvite}
+                    onChange={() => { appState.emitRaceSettings(appState.myPlayerID, appState.enableInvite); }}
```

Comment:
run formatting here should be able to be simplified to

`onChange={() => appState.emitRaceSettings(appState.myPlayerID, appState.enableInvite)}`